src/database/index.tsx

```diff
@@ -605,6 +617,26 @@ const batchSetFacilityModelVersions = (
   });
 };
 
+const fetchFacilityAndVersions = async (
+  facilityRef: firebase.firestore.DocumentReference<
```

Comment:
Nit: I'd put `scenarioId` before `facilityRef` in arg order because 1) it matches with the top-down hierarchy of the object model, and 2) it is more consistent with other function signatures in nearby code.